### PR TITLE
Feature: allow decision editing

### PR DESCRIPTION
**Pull request by miggol**
This addresses #638 

Besides restoring the editing link, reviewers now get a warning if they are making the final decision that will cause the review to enter the `CLOSING` stage. For in that case, they won't be able to edit their response after the fact.

Note: this branch started in acceptation, but I think it's more suited to go through develop. Therefore, this has some extra commits to be backmerged.

### CODE REVIEW

File: reviews/templates/reviews/decision_form.html [hunk 1]
```diff
@@ -24,6 +24,13 @@ <h2>{% trans "Aanvraag beoordelen" %}</h2>
                             De aanvraag is <a href="{{ pdf_url }}" target="_blank">hier</a> in te zien (downloadt als PDF).
                         {% endblocktrans %}
                     </p>
+                    {% if decision.is_final_decision %}
+                        <p>
+                            {% blocktrans trimmed %}
+                                <strong>Attentie:</strong> jij bent de laatste beoordelaar van deze aanvraag. Daarom kun je na het versturen je oordeel en commentaar niet meer bewerken. Controleer je antwoorden goed.
+                            {% endblocktrans %}
```
Comment (tymees) [inline reviews/templates/reviews/decision_form.html:31]: You might want to consider using an `alert` box instead. To draw more attention to it.

File: locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -3731,10 +3738,14 @@ msgid "Commissieleden aanstellen"
 msgstr "Appoint committee members"
 
 #: reviews/templates/reviews/review_assign_form.html:31
-#, python-format
+#, fuzzy, python-format
+#| msgid ""
+#| "<p> Kies hier de geschikte route en commissieleden voor de aanvraag "
+#| "<em>%(title)s</em>. klik <a href=\"%(workload_url)s\">hier</a> voor een "
+#| "overzicht van de werkverdeling van deze commissie. </p>"
```
Comment (tymees) [inline locale/en/LC_MESSAGES/django.po:3745]: Fuzzy! Please check if you need to resolve this and remove the fuzzy bit if not

**Review by tymees — CHANGES_REQUESTED** [review]
I've only reviewed your commits, and ignored the to-backport commits. 

Looks good, simple but effective. 

The only two commits are very minor, so when you've looked at those you can consider it approved.

File: reviews/templates/reviews/decision_form.html (hunk 1 above)
Comment (miggol) [reply reviews/templates/reviews/decision_form.html:31]: An alert does look better, good idea

File: locale/en/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -3743,7 +3746,10 @@ msgid ""
 "<p> Kies hier de geschikte route en commissieleden voor de aanvraag "
```
Comment (EdoStorm96) [inline locale/en/LC_MESSAGES/django.po:3746]: I think you need to remove `fuzzy, ` here (line 3744), so that this will compile correctly.

**Review by EdoStorm96 — CHANGES_REQUESTED** [review]
Nicely done overall. I have one minor comment and one other suggestion. I think it would be nice if the text for the decide action changes, depending on whether you've already taken a decision. So in `DecideAction.description()` maybe you an add something like:
```
if decision.go == "":
    return _("geef jouw beslissing en/of commentaar door")
return _("Pas jouw beslissing en/of commentaar aan")
```

File: locale/en/LC_MESSAGES/django.po (hunk 3 above)
Comment (miggol) [reply locale/en/LC_MESSAGES/django.po:3746]: Ah yes, I was getting used to `po-mode` in emacs which apparently doesn't do that automatically. Thanks

**Pull request merged**